sdks/python/src/opik/evaluation/metrics/llm_judges/moderation/metric.py

```diff
@@ -11,6 +11,11 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class ModerationResponseFormat(BaseModel):
+    score: int
```

Comment:
Current template asks the model to give a float in range [0, 1], so there is a serious contradiction.
Option 1: we should change type to "float" here. (faster way)
Option 2: we should update the template and ask the model to provide integers in range [0, 10] instead. `_parse_model_output` method will need to be updated as well then. The possible motivation to go this way can be that models give better outputs when you ask them to provide 0-10 scores instead of 0-1 (that is what @japdubengsub told me after working on g-eval metric and reading the paper).

I haven't checked other metrics, but the same comment might be relevant for other response formats in this PR.

Reply:
Good catch, should be a float here. Not sure I've seen the issue with asking for a number between 0 and 10 instead of 0 to 1, I think we can keep it like this for now